example/cumulus-tf/ingest_and_publish_granule_workflow.asl.json

```diff
@@ -237,6 +237,32 @@
         }
       ]
     },
+    "UpdateGranulesCmrMetadataFileLinksStep": {
```

Comment:
We need to update all the other workflow definitions that use `MoveGranules` (e.g. `cookbook_browse_example_workflow.asl.json`) to use your new task as well, especially because these workflow definitions are referred to elsewhere in the docs so they need your new task to function the same as they did before

Reply:
Okay, will update

Reply:
@markdboyd I have updated `example/cumulus-tf/cookbook_browse_example_workflow.asl.json` https://github.com/nasa/cumulus/pull/1983/files#diff-327db09f4ba63435d3702da023c4b92138b6cccae186407c9770f2ff3d8bbe79

Reply:
I think there are other workflows in this directory using `MoveGranules`. You might want to do a search in this directory